hashicorp/web-platform-packages
# feat(plugin): Dato Table Editor Plugin

🎟️ [Asana Task](https://app.asana.com/0/1200531701094631/1202533327481359/f)
🎨 [Design](https://www.figma.com/file/uxnWWpM14UcttJNvBadRfL/Pricing?node-id=205%3A7920)
^ Motivation for plugin

---

## Description



This PR includes a plugin for DatoCMS to enter our pricing/features data. It is built on top of a currently published [plugin](https://www.datocms.com/marketplace/plugins/i/datocms-plugin-table-editor?s=table) and includes different types of cell inputs including a wysiwyg editor with both heading and description, as well as a checkbox. The table also includes options to hide column headers and set rows to be collapsible. See [readme](https://github.com/hashicorp/web-platform-packages/pull/75/files#diff-5af5f7ff8d365754a8c4d3a2d3499d13b3b94e7e58a0ad3cd460fdd63218318f) for more information.

Once merged,  the plugin will be published and available in the [DatoCMS Plugin Marketplace](https://www.datocms.com/marketplace/plugins)

It is in the 🛹 stage and could use style updates, code refactoring, tests, etc.

### Testing instructions:
`cd packages/datocms-plugin-table`
`npm start`
Edit in [dato](https://io-marketing-pages.admin.datocms.com/editor/item_types/1926417/items/116196824). See **FOR TESTING ONLY - Features Table** section at the bottom of the page.

## PR Checklist 🚀

- [ ] Write a useful description (above) to give reviewers appropriate context.
